app/src/pages/playground/playgroundUtils.ts

```diff
@@ -189,14 +190,32 @@ function processAttributeMessagesToChatMessage({
   provider: ModelProvider;
 }): ChatMessage[] {
   return messages.map(({ message }) => {
+    if (Array.isArray(message.content)) {
+      // check if message matches anthropic tool result schema
+      const { data, success } =
+        anthropicToolResultMessageSchema.safeParse(message);
+      const toolResult = data?.content.find(
+        (content) => content?.type === "tool_result"
+      );
+      if (success && toolResult != null) {
+        return {
+          id: generateMessageId(),
+          role: getChatRole("tool"),
+          content: toolResult.content,
```

Comment:
so bascially we are just finding the tool result content (which is a string) and plugging that into the message content to not have to overload it?

Reply:
sounds good